README.md

```diff
@@ -36,9 +59,27 @@ verbing('swimming')
 verbing('go')
 // =>  'go'
 ```
- 
+//Answer to 6 lines 63-74-??? Not clear
+let str = "dock"
+let atLeastThree= "ing"
+let alreadyEnding = "ly"
+
+if (str.length >= 3){
```

Comment:
This code doesn't run properly as there are problems with curly brackets. Proper syntax would be this: 
let str = "ing"
let atLeastThree= "ing"
let alreadyEnding = "ly"

if (str.length >= 3){
  console.log(str + atLeastThree)
} else {
    if (str + atLeastThree) {
      console.log(str + "ly")
        }
if (str.length < 3) {
  console.log(str)
}
}